samples/drivers/led_ws2812/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Is this file really required? Boards that want to use SPI driver enable it in their respective `.conf` files and for GPIO based bit-bang driver SPI shouldn't be enabled.

Reply:
Why would boards want to have logic for this sample in their board files?

My guess is most people will wire an Adafruit ring to an existing dev board. I don't know of any boards with built-in rings.